helios-system-tests/src/main/java/com/spotify/helios/system/MultiplePortJobTest.java

```diff
@@ -92,14 +92,22 @@ public void test() throws Exception {
 
       final JobId jobId2 = createJob(testJobName + 2, testJobVersion, BUSYBOX, IDLE_COMMAND,
                                      EMPTY_ENV, ports2);
-
       assertNotNull(jobId2);
       deployJob(jobId2, testHost());
       final TaskStatus firstTaskStatus2 = awaitJobState(client, testHost(), jobId2, RUNNING,
                                                         LONG_WAIT_SECONDS, SECONDS);
 
-      assertEquals(expectedMapping1, firstTaskStatus1.getPorts());
-      assertEquals(expectedMapping2, firstTaskStatus2.getPorts());
+      // Verify we allocated dynamic ports from within the specified range.
+      assertTrue(portRange.contains(firstTaskStatus1.getPorts().get("foo").getExternalPort()));
+      assertTrue(portRange.contains(firstTaskStatus2.getPorts().get("foo").getExternalPort()));
```

Comment:
not really important but a small tip: I've taken to writing checks like this using assertThat instead, like

```java
assertThat(portRange, hasEntry(firstTaskStatus2.getPorts().get("foo").getExternalPort()));
```

because when it fails, junit will give you an error message like "Expected {4711, 4722} to have entry 1234" (or whatever), whereas `assertTrue(something that is false)` just reports an empty and unhelpful AssertionError